### PR TITLE
styles

### PR DESCRIPTION
**Pull request by NicoleBrady**

### CODE REVIEW

File: src/Sass/_variables.scss [hunk 1]
```diff
@@ -47,7 +49,7 @@ $background: #e3f4f3;
 }
 
 @mixin darkBoxShadow {
-    box-shadow: black 15px 15px;
+    box-shadow: #e6e6e6 15px 15px;
```
Comment (Edjrich) [inline src/Sass/_variables.scss:52]: 😎

**Review by michellelven — APPROVED** [review]
It's starting to look real pretty!! 🌷 🌼

File: src/Start.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Link } from "react-router-dom";
 
 const Start = () => {
     return (
-        <div>
+        <div class="gameContainer">
```
Comment (Edjrich) [inline src/Start.js:5]: this name might be confusing since it's in Start.js, not Game.Js

**Review by Edjrich — APPROVED** [review]

**Review by AHeroOfTime — APPROVED** [review]
👍

**Pull request merged**